composer.json

```diff
@@ -17,7 +17,8 @@
 	],
 	"require": {
 		"php": ">=5.3.3",
-		"composer/installers": "~1.0"
+		"composer/installers": "~1.0",
+		"monolog/monolog": "~1.15"
```

Comment:
that's at least 1.15, e.g. 1.20 or 1.9999 (theoretically).

~ means "at least, but only allowed to increment the last digit"